package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "tech-study-finder",
+  "private": true,
+  "workspaces": [
+    "client",
+    "server"
+  ],
+  "scripts": {
+    "dev": "concurrently \"npm start --workspace server\" \"npm start --workspace client\"",
+    "lint": "eslint .",
+    "format": "prettier --write .",
+    "prepare": "husky install",
```

```suggestion
    "prepare": "npx husky install",
```
"husky install" allein hat bei mir nicht funktioniert

---

`sh: husky: command not found` Das war der Fehler den ich bekommen habe bei `npm install`
das wird beim prepare Skript ausgeführt aber (wenn ich es richtig verstanden habe) funktioniert das nur, wenn husky schon im PATH ist. Vielleicht war es aber auch nur bei mir so :)

---

Super danke! Müssen ja eh alle mal ausprobieren